core/variant/callable.cpp

```diff
@@ -37,7 +37,7 @@
 #include "core/object/script_language.h"
 
 void Callable::call_deferredp(const Variant **p_arguments, int p_argcount) const {
-	MessageQueue::get_singleton()->push_callablep(*this, p_arguments, p_argcount);
+	MessageQueue::get_singleton()->push_callablep(*this, p_arguments, p_argcount, true);
```

Comment:
This makes the deferred calls print error when they fail, instead of doing so silently.

Using `callable_mp()` can be very error prone, so this helps finding wrong calls. I just wonder if it has unforeseen side effects.